api/src/services/external/apiNodeService.ts

```diff
@@ -272,18 +279,18 @@ export async function getProposal(id: number) {
   try {
     const { data } = await axios.get<CosmosGovProposalResponse>(`${apiNodeUrl}/cosmos/gov/v1beta1/proposals/${id}`);
 
-    const proposer = null; // TODO: Fix
-    if (id > 3) {
-      // const proposerResponse = await fetch(`${apiNodeUrl}/gov/proposals/${id}/proposer`);
-      // const proposerData = await proposerResponse.json();
-      // const validatorFromDb = await Validator.findOne({ where: { accountAddress: proposerData.result.proposer } });
-      // proposer = {
-      //   address: proposer,
-      //   moniker: validatorFromDb?.moniker,
-      //   operatorAddress: validatorFromDb?.operatorAddress,
-      //   avatarUrl: validatorFromDb?.keybaseAvatarUrl
-      // };
-    }
+    // const proposer = null; // TODO: Fix
```

Comment:
is there a reason we can't remove this code? I know it's here only because of prettier so just asking

Reply:
We have kept it in hope of fixing it one day. There is an issue with proposals created during genesis. I created #132 to keep track of it.